plugins/search/providers/elastic_search/lib/model/filters/ESearchQueryFromAdvancedSearch.php

```diff
@@ -100,6 +103,25 @@ protected function createESearchQueryFromEntryCatpionAdvancedFilter(kEntryCaptio
 		return $result;
 	}
 
+	protected function createESearchQueryFromEntryQuizAdvancedFilter(kQuizAdvancedFilter $filter)
+	{
+		$item = new ESearchEntryItem();
+		$item->setFieldName(ESearchEntryFieldName::IS_QUIZ);
+		$item->setItemType(ESearchItemType::EXISTS);
+		if($filter->getIsQuiz())
+		{
+			$result = $item;
+		}
+		else
+		{
+			$result = new ESearchOperator();
```

Comment:
Since the NOT option repeat itself, and will repeat more, lets consolidate it into a smaller function - 
$query = createNegativeQuery(array $items,$operator=null)
And in your case you can call it -
$item =  createNegativeQuery(array($item));